test/claim-exp.test.js

```diff
@@ -196,28 +196,22 @@ describe('expires', function() {
       expect(decoded.exp).to.equal(60);
     });
 
-    // TODO an exp of -Infinity should fail validation
-    it('should set null "exp" when given -Infinity', function () {
-      const token = signWithExpiresIn({exp: -Infinity});
-
-      const decoded = jwt.decode(token);
-      expect(decoded.exp).to.be.null;
+    it('should throw exception when given -Infinity', function () {
+      expect(() => signWithExpiresIn({exp: -Infinity})).to.throw(
+        '"exp" should be a number of seconds'
+      );
     });
 
-    // TODO an exp of Infinity should fail validation
-    it('should set null "exp" when given value Infinity', function () {
-      const token = signWithExpiresIn({exp: Infinity});
-
-      const decoded = jwt.decode(token);
-      expect(decoded.exp).to.be.null;
+    it('should throw exception when given Infinity', function () {
+      expect(() => signWithExpiresIn({exp: Infinity})).to.throw(
+        '"exp" should be a number of seconds'
+      );
```

Comment:
Ditto to the above comment, this test case can be moved into the array in the `'`jwt.sign` "exp" claim validation'` block.